files/templates/scrapeconfigs/additional-scrape-configs.template.yaml

```diff
@@ -255,22 +255,32 @@
   relabel_configs:
   - source_labels: [__address__]
 [[- if .CAPICluster ]]
-    replacement: $1:10257
+    replacement: 10257
 [[- else ]]
-    replacement: $1:10252
+    replacement: 10252
 [[- end ]]
+    target_label: __port
```

Comment:
Would it make sense to rename it to __tmp_port ?

Also do not forget to use a label drop to remove the label at the end

Reply:
Ok we rename by __tmp_port.

BTW, label drop is not necessary :
"Labels starting with __ will be removed from the label set after target relabeling is completed." [prometheus doc](https://prometheus.io/docs/prometheus/latest/configuration/configuration/)